tests/acceptance/bootstrap/OcisConfigContext.php

```diff
@@ -120,6 +120,33 @@ public function theAdministratorHasEnabledTheRole(string $role): void {
 		$this->setEnabledPermissionsRoles($defaultRoles);
 	}
 
+	/**
+	 * @Given the administrator has enabled the following permissions roles:
```

Comment:
```suggestion
	 * @Given the administrator has enabled the following permission roles:
```

Reply:
`permissions role` is standard we followed. Other similar steps have the same.